newIDE/app/src/Debugger/index.js

```diff
@@ -107,12 +105,6 @@ export default class Debugger extends React.Component<Props, State> {
     }
   }
 
-  componentWillReceiveProps(nextProps: Props) {
```

Comment:
What about the component did mount? If isActive is false at the mounting of the component, then the server callbacks will never be added.
I guess we should then remove entirely isActive prop from this component if it's not used anymore.